catalog/to-k8s/source_test.go

```diff
@@ -125,31 +151,41 @@ func TestSource_ignoreK8S(t *testing.T) {
 	})
 
 	expected := map[string]string{
-		"consul": "consul.service.test",
-		"svcA":   "svcA.service.test",
+		"consul": "consul.service.ignoreK8S",
+		"svcA":   "svcA.service.ignoreK8S",
```

Comment:
Running locally I got:
```
Expected :map[string]string{"consul":"consul.service.ignoreK8S", "svcA":"svcA.service.ignoreK8S"}
Actual   :map[string]string{"svcA":"svcA.service.ignoreK8S"}
```

I know your change didn't cause this but I think it's a small fix. The `len` check just above should check that len != 2, or you could even just put the whole assertion in there:
```go
	var actual map[string]string
	retry.Run(t, func(r *retry.R) {
		sink.Lock()
		defer sink.Unlock()
		actual = sink.Services
		if len(actual) == 0 {
			r.Fatal("services not found")
		}
	})

```